src/csharp/Microsoft.Spark/Sql/SparkSession.cs

```diff
@@ -101,6 +101,44 @@ public void Dispose()
         public DataFrameReader Read() =>
             new DataFrameReader((JvmObjectReference)_jvmObject.Invoke("read"));
 
+        /// <summary>
+        /// Creates a DataFrame with a single column named id, containing elements in
+        /// a range from 0 to end (exclusive) with step value 1.
+        /// </summary>
+        /// <returns>DataFrame object</returns>
+        public DataFrame Range(long end) =>
+            new DataFrame((JvmObjectReference)_jvmObject.Invoke("range", end));
+
+        /// <summary>
+        /// Creates a DataFrame with a single column named id, containing elements in 
+        /// a range from start to end (exclusive) with step value 1.
+        /// </summary>
+        /// <returns>DataFrame object</returns>
+        public DataFrame Range(long start, long end) =>
+            new DataFrame(
+                (JvmObjectReference)_jvmObject.Invoke("range", start, end));
```

Comment:
Looks like this can fit in one line without going over 100 limit?
```C#
            new DataFrame((JvmObjectReference)_jvmObject.Invoke("range", start, end));
```

Reply:
yes sorry i had resharpers parameter hints and didnt realise they caused vs to show the length including the hints